Server/Test_Plan_OCIS_Files_Texteditor.md

```diff
@@ -0,0 +1,21 @@
+# Markdown Editor
+
+Test Case
+
+| Test Case |  Expected result | Result  | Related Comment |
+| :---------| :-------------- | :------:| --------------- |
+| Toggle (fullscreen in page,fullscreen,preview,previewonly,html preview,catalog)  | It works |  | |
+| Paste a URL link and click it | URL should open in new tab (user should not loose any progress)|  | |
+| No external links should be present eg. github link | no external link |  | |
+| Add an markdown image manually `![alt](url)` | image should render |  | |
+| Upload image in editor (add image link,upload image,clip upload) | image should render |  | |
+| Use inline and block formulas (`$x+y^{2x}$`,`$$ \sqrt[3]{x} $$`) | formulas should render correctly |  | |
+| Create a mermaid diagramming | Mermaid diagramming is rendered correctly|  | |
+| Click save button | progress should be saved and user should recive saved feedback (message or visual indication) |  | |
```

Comment:
```suggestion
| Click save button | progress should be saved and user should receive saved feedback (message or visual indication) |  | |
```

Reply:
I could not. Or maybe I just misunderstood what the issue is :shrug: